client/src/test-runner/runner.ts

```diff
@@ -166,80 +162,73 @@ export class ElmTestRunner {
     return true;
   }
 
-  async fireDecorationEvents(
-    suite: TestSuiteInfo,
-    testStatesEmitter: vscode.EventEmitter<
-      TestRunStartedEvent | TestRunFinishedEvent | TestSuiteEvent | TestEvent
-    >,
-  ): Promise<number> {
-    const testInfosByFile = getTestInfosByFile(suite);
-    return Promise.all(
-      Array.from(testInfosByFile.entries()).map(([file, nodes]) => {
-        return vscode.workspace
-          .openTextDocument(file)
-          .then((doc) => {
-            const text = doc.getText();
-            return nodes.map((node) => {
-              const id = node.id;
-              const event = this.eventById.get(id);
-              if (!event) {
-                throw new Error(`missing event for ${id}`);
-              }
-              const names = event.labels.slice(1);
-              const offset = findOffsetForTest(
-                names,
-                text,
-                (offset) => doc.positionAt(offset).character,
-              );
-              const line = (offset && doc.positionAt(offset).line) ?? 0;
-              const lineOf = (search: string) => {
-                const index = text.indexOf(search, offset);
-                return index >= 0 ? doc.positionAt(index).line : line;
-              };
-              return createTestEvent(id, event, lineOf, line);
-            });
-          })
-          .then(
-            (events) =>
-              events.map((event) => {
-                testStatesEmitter.fire(event);
-                return true;
-              }).length,
-          );
-      }),
-    ).then((counts) =>
-      counts.length > 0 ? counts.reduce((a, b) => a + b) : 0,
-    );
-  }
-
-  async runAllTests(): Promise<TestLoadFinishedEvent> {
-    this.loadedSuite = undefined;
-    return this.runSomeTests();
-  }
-
-  getFilesAndAllTestIds(tests: string[]): [string[], string[]] {
-    if (!this.loadedSuite) {
-      throw new Error("not loaded?");
-    }
-    return getFilesAndAllTestIds(tests, this.loadedSuite);
-  }
-
-  async runSomeTests(files?: string[]): Promise<TestLoadFinishedEvent> {
-    return new Promise<TestLoadFinishedEvent>((resolve) => {
+  // async fireDecorationEvents(
```

Comment:
Needs to be removed before we merge this.

Reply:
Done